deck.gl__layers/index.d.ts

```diff
@@ -710,12 +710,12 @@ declare module "@deck.gl/layers/polygon-layer/polygon-layer" {
 	import { RGBAColor } from "@deck.gl/core/utils/color";
 	export interface PolygonLayerProps<D> extends CompositeLayerProps<D> {
 		filled?: boolean;
-		stroked: boolean;
-		extruded: boolean;
+		stroked?: boolean;
+		extruded?: boolean;
 		wireframe?: boolean;
 		elevationScale?: number;
-		lineWidthUnits?: string;
-		lineWidthScale?: boolean;
+		lineWidthUnits?: "meters" | "pixels";
```

Comment:
Would you mind creating a new type for Units? Since we have it in 3 other places

Reply:
Absolutely, sorry about that.

Reply:
no worries, thanks for the fixes!